lib/hydra_public_api_client.rb

```diff
@@ -15,7 +14,7 @@ def fetch_oauth_user(...)
   def fetch_oauth_user(token:)
     if bypass_auth?
       users = stubbed_data['users']
-      user = users.detect { |attr| attr['id'] == BYPASS_AUTH_USER_ID }
+      user = users.detect { |attr| attr['id'] == '00000000-0000-0000-0000-000000000000' }
```

Comment:
Minor: I wonder whether there's _some_ value in retaining the `BYPASS_AUTH_USER_ID` constant since it does make it a bit clearer what it's purpose is...

Reply:
I'm planning to do some more work around this so I'm going to leave it as-is for now.